tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -418,13 +418,18 @@ def test_is_json_compatible(self):
                     ]), True), True)
         ])
 
-        valid_schemas = [message_schema, prompt_response_schema, combined_schema]
+        schema_with_string_map_keys = StructType(
```

Comment:
```suggestion
        string_map_keys_schema = StructType(
```
just to make variable naming the same as the other schemas